ntoskrnl/mm/ARM3/sysldr.c

```diff
@@ -2964,7 +2964,52 @@ MmLoadSystemImage(IN PUNICODE_STRING FileName,
     PrefixName = *FileName;
 
     /* Check if we have a prefix */
-    if (NamePrefix) DPRINT1("Prefixed images are not yet supported!\n");
+    if (NamePrefix)
+    {
+        /* Check if "directory + prefix" is too long for the string */
+        if(PrefixName.MaximumLength + BaseDirectory.Length < PrefixName.MaximumLength)
```

Comment:
```suggestion
        if (PrefixName.MaximumLength + BaseDirectory.Length < PrefixName.MaximumLength)
```

Reply:
Can you please leave the style nitpicking for the last iteration, it makes the entire PR unreadable if you add a comment on every line...

Reply:
The if statement styles is changed to the proposed style now.